lib/java/collections.nit

```diff
@@ -46,6 +46,48 @@ extern class AbstractJavaArray[E: Object]
 	redef fun reverse_iterator do return new JavaArrayReverseIterator[E](self)
 end
 
+#Java primitive array `int[]`
```

Comment:
To be consistent with the style of the module, there's a space missing between `#J`.